src/big/big_decimal.cr

```diff
@@ -207,6 +232,10 @@ struct BigDecimal < Number
     end
   end
 
+  def round
```

Comment:
Why is this not a default argument on `round(Int)`?

Reply:
You mean *digits* arguments in `Number#round(digits, base = 10)`?

Reply:
I can add it, I'm not sure though should it be done in this PR...

Reply:
I don't think anything speaks against having a default argument in `Number#round`. Should better be a separate PR.

Reply:
Pushed #5397 with that, cheers ❤️